README.md

```diff
@@ -445,6 +445,7 @@ Tells Conan where to install the package. It is similar to specifying `CMAKE_INS
 The following `options` are available to pass on `conan install` when building the oneMKL library:
 
 - `build_shared_libs=[True | False]`. Setting it to `True` enables the building of dynamic libraries. The default value is `True`.
+- `target_domains=[<any value>]`. Setting it to `blas` or any other domain, enables building of that specific domain only. If not defined, the default value is all supported domains.
```

Comment:
Does conan support only one or all domains (seems that cmake supports list of domains)?

Reply:
Conan does not care what option is provided. IT will just pass it to CMake. CMake will then perform checks on the value as per the changes. I will update this section to reflect that a list is supported. Thanks